src/constans/VALIDATIONS.ts

```diff
@@ -0,0 +1,22 @@
+export const emailValidation = {
+  required: "Email is required",
+  pattern: {
+    value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
+    message: "Please enter a valid email",
+  },
+};
+
+const RegExp = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[\W_]).{4,}$/;
```

Comment:
We can rename this to passwordRegEx. As we can have multiple regular expressions. For regular expressions, it is also recommended to add a comment descripting what they do as they are difficult to understand by default.